models/base.go

```diff
@@ -262,3 +262,27 @@ func isValidMetricColumn(name string) bool {
 	_, isValid := fields[name]
 	return isValid
 }
+
+func agentTypeToClickHouseEnum(agentTypeName string) string {
+	// AgentTypes represents Agent type as stored in database.
+	// Should be same as in pmm/inventorypb/agents.proto
```

Comment:
Please use `inventorypb.AgentType` and constants like `invetorypb.AgentType_QAN_MYSQL_PERFSCHEMA_AGENT`. That way compiler can check them.

Reply:
Changed.